spec/features/mediated_deposit_versioning_spec.rb

```diff
@@ -45,6 +45,8 @@
 
       # A work submitted for approval should not be editable.
       expect(page).not_to have_css("a[aria-label='Edit #{new_work_title}']", wait: 0)
+      sign_out
+      sleep(1)
```

Comment:
Just `sign_out` didn't work, so added sleep.

Reply:
Can you add a comment about why the sleep is necessary? Otherwise I foresee someone removing this for lack of understanding why it was here.

Reply:
Done.